environment.cpp

```diff
@@ -69,6 +69,17 @@ std::vector<double> create_n_inputs(int n_inputs)
   return input_vector;
 }
 
+std::vector<double> create_n_inputs(environment e, const int &n_inputs, std::mt19937_64 &rng)
+{
+  std::vector<double> input_vector(n_inputs);
+
+  for(auto& cue : input_vector){
+      cue = e.get_dist()(rng);
+    }
```

Comment:
Nice! I would have created an empty vector and pushed_back() new elements with a for loop that iterates `n_inputs` times, but I like this method, and its actually faster! (not that it makes a big difference here)